src/database/datatools/nomass_materials.py

```
@@ -0,0 +1,146 @@
+                            thermal_absorptance: Optional[float] = None,
+                            solar_absorptance: Optional[float] = None,
+                            visible_absorptance: Optional[float] = None) -> None:
+    
+    conn = sqlite3.connect(db_path)
+    conn.row_factory = sqlite3.Row
+    cursor = conn.cursor()
+    cursor.execute("SELECT * FROM no_mass_materials WHERE id = ?", (material_id,))
+    row = cursor.fetchone()
+    if row is None:
+        conn.close()
+        raise ValueError(f"No_mass material with id {material_id} does not exist.")
```

Ruff (0.15.2)

[warning] Avoid specifying long messages outside the exception class

(TRY003)